### DISCUSSION TITLE
Medical Images into CGNS format

### DISCUSSION THREAD

**Discussion by priyambadapraharaj** [CGNS/CGNS#289] [CGNStalk]
Can we convert medical images(which are in NIFTI or DICOM format) to .CGNS format.

**Comment by MicK7** [comment]
What is the content of a NIFTI or DICOM file ?
As you mention "images", CGNS format is not optimized to store regularly spaced data like dedicated images' file format. CGNS requires a support mesh. With few indication provided this seems like the only obvious limitation.

**Comment by brtnfld** [comment]
HDF5 can store images

https://support.hdfgroup.org/HDF5/Tutor/h5image.html

but currently, CGNS does not handle images in this standard way. The CGNS standard would have to be extended via a CPEX to handle images according to a standard.
